src/h5web/providers/mock/data.ts

```diff
@@ -64,6 +55,9 @@ export const mockMetadata = makeNxGroup(mockDomain, 'NXroot', {
                 id: 'title_twoD',
               }),
             }),
+            makeNxGroup('absolute_default_path', 'NXentry', {
+              defaultPath: '/nexus_entry/nx_process/nx_data',
+            }),
```

Comment:
Since `supportsEntity` no longer checks for a default path, I had to remove a few unit tests. Most of them were testing things already covered by feature tests, but this was the exception, so I've added this group here as well as a corresponding feature test. Same goes with `default_not_string` below.